pkg/module/reporter/htmlreporter/htmlreporter.go

```diff
@@ -59,6 +60,12 @@ func (r *HTMLReporter) Configure(config map[string]string) error {
 		r.baseURL = "file:///var/lib/qmstr/reports"
 	}
 
+	if statics, ok := config["generatestatics"]; ok && statics == "no" {
```

Comment:
I think "generatestatics" is not a good name for the option. The user knows that we generating HTML, not that it is static HTML. I guess it is better to call it "generatehtml".